ui/src/app/utils/runtime-utils.tsx

```diff
@@ -681,10 +682,7 @@ export const withAnalysisConfigDefaults = (
   let existingDiskName = null;
   const computeType = r.computeType ?? ComputeType.Standard;
   if (computeType === ComputeType.Standard) {
-    dataprocConfig = null;
-    if (detachable === false) {
-      detachableType = null;
-    } else if (detachable === true || existingDisk) {
+    if (detachable === true || existingDisk) {
```

Comment:
Please add a comment to the function `withAnalysisConfigDefaults()` describing what this function does and how it is used - let's help future maintainers not have to go through what we're doing again